src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -275,70 +314,80 @@ public Style Style
 
 		#endregion
 
-		protected virtual void OnStyleChanged(Style oldStyle, Style newStyle)
+		private void OnStyleChanged(Style oldStyle, Style newStyle)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'newStyle'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)